backend/tracim_backend/views/core_api/workspace_controller.py

```diff
@@ -401,19 +401,26 @@ def create_workspaces_members_role(
         try:
             _, user = uapi.find(
                 user_id=hapic_data.body.user_id,
-                email=hapic_data.body.user_email_or_public_name,
-                public_name=hapic_data.body.user_email_or_public_name
+                email=hapic_data.body.user_email,
+                public_name=hapic_data.body.user_public_name
             )
             if user.is_deleted:
                 raise UserIsDeleted('This user has been deleted. Unable to invite him.')  # nopep8
             if not user.is_active:
                 raise UserIsNotActive('This user is not activated. Unable to invite him')  # nopep8
         except UserDoesNotExist as exc:
-            if not app_config.EMAIL_NOTIFICATION_ACTIVATED:
+            # INFO - G.M - 2018-10-25 - do not allow simple user
```

Comment:
this behavior should be in core api, I think, not in controller

Reply:
@lebouquetin  something like uapi.allowed_to_invite_user(email) which care of all case (no email, mail notification disabled, simple user) ? or just unallow simple user in uapi.create_user() ?

Reply:
I have no specific idea. Do what you feel being the most robust / evolutive / easier to use in another context (for example by creating an equivalent CLI command)